test/addons/rook-cephfs/provision-test/pvc.yaml

```diff
@@ -12,4 +12,4 @@ spec:
   resources:
     requests:
       storage: 1Gi
-  storageClassName: rook-cephfs
+  storageClassName: rook-cephfs-test-fs1
```

Comment:
Maybe also pass filesystem name as parameter?

Reply:
i had thought about it earlier. I was thinking it would spin more changes. but then again it's just one file. I will make the change.